services/common/changeRequest.js

```diff
@@ -268,15 +268,23 @@ export const CONFIG = {
     amendmentTransmittalNumber: {
       ...waiverBaseTransmittalNumber,
       idFieldHint: [
-        { text: "Must follow the format SS.####.R##.M## or SS.#####.R##.M##" },
+        { text: "Must follow the format SS-####.R##.## or SS-#####.R##.##" },
       ],
-      idFormat: "SS.####.R##.M## or SS.#####.R##.M##",
-      idRegex: "^[A-Z]{2}[.][0-9]{4,5}[.]R[0-9]{2}[.]M[0-9]{2}$",
+      idFormat: "SS-####.R##.## or SS-#####.R##.##",
+      idRegex: "^[A-Z]{2}[-][0-9]{4,5}[.]R[0-9]{2}[.](0[1-9]|[1-9][0-9])$",
```

Comment:
I like that you used the regex to invalidate .00 as a possible amendment number :)